projects/ngrx.io/src/app/app.component.spec.ts

```diff
@@ -370,11 +371,12 @@ describe('AppComponent', () => {
         expect(component.showSearchResults).toBe(true);
       });
 
-      it('should show developer source view if the footer is clicked while holding the meta and alt keys', () => {
-        component.dtOn = false;
-        expect(component.onClick({ tagName: 'FOOTER'} as HTMLElement, 0, false, true, true)).toBeFalsy();
-        expect(component.dtOn).toBeTruthy();
-      });
+      // causing issues with aio-search-box nested in ng-content in aio-top-menu
```

Comment:
Instead of commenting out the test, just exclude it

Reply:
Done